src/_includes/header.html

```diff
@@ -1,42 +1,37 @@
 <header>
-  <nav>
-    <picture>
-      <a href="/">
-        <img src="images/calitp_logo_MAIN.png" alt="Cal-ITP: California Integrated Travel Project" width="215" />
+  <nav class="navbar navbar-expand-lg fixed-top bg-white">
+    <div class="container">
+      <a class="navbar-brand" href="/">
+        <img src="/images/calitp_logo_MAIN.png" alt="Cal-ITP: California Integrated Travel Project" width="160" />
```

Comment:
🙏 Thank you! This is broken in the `feat/press-collection` branch because of the relative URL, but going to leave it alone since you are working on it here.